### PR TITLE
Move producer to knative-serving by creating new local service

### PR DESCRIPTION
**Pull request by beemarie**

### CODE REVIEW

**Comment by beemarie** [comment]
Closes #33

File: cmd/controller/main.go [hunk 1]
```diff
@@ -23,5 +24,6 @@ import (
 func main() {
 	sharedmain.Main("async-controller",
 		ingress.NewController,
+		service.NewController,
```
Comment (julz) [inline cmd/controller/main.go:27]: fwiw Im not sure you'll need a separate reconciler for this. The "kind" you're reconciling is still the top-level ingress, you're just going to spit out a Service as well as delegate-ingress, iiuc

Comment (beemarie) [reply cmd/controller/main.go:27]: I went back and forth on this. I think you're right, it's actually cleaner to keep one reconciler.

**Review by maximilien — APPROVED** [review]
LGTM

**Review by julz — APPROVED** [review]
/lgtm

**Pull request merged**